wagtail/admin/static_src/wagtailadmin/js/modal-workflow.js

```diff
@@ -51,22 +54,36 @@ function ModalWorkflow(opts) {
         });
     };
 
-    self.loadResponseText = function(responseText) {
-        var response = eval('(' + responseText + ')');
+    self.loadResponseText = function(responseText, textStatus, xhr) {
+        /* RemovedInWagtail24Warning - support for eval()-ing text/javascript responses
```

Comment:
I would drop it now. It is an internal API. People who have decided to use it, knew what they were doing.